utplsql-maven-plugin/src/test/java/org/utpsql/test/UtPLSQLMojoTest.java

```diff
@@ -0,0 +1,35 @@
+package org.utpsql.test;
+
+import org.apache.maven.plugin.testing.MojoRule;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.utplsql.UtPLSQLMojo;
+
+public class UtPLSQLMojoTest
+{
+	public static final String POM_PATH = "src/test/resources/pom.xml";
+
+	@Rule
+	public MojoRule rule = new MojoRule()
+	{
+		@Override
+		protected void before() throws Throwable
+		{
+		}
+
+		@Override
+		protected void after()
+		{
+		}
+	};
+
+	@Test
+	public void testDefinition() throws Exception
+	{
+		UtPLSQLMojo myMojo = (UtPLSQLMojo) rule.lookupMojo("test", POM_PATH);
+		Assert.assertNotNull(myMojo);
+		myMojo.execute();
```

Comment:
You check only execution itself, right? Would be great to test the results, too

Reply:
Right, we are going to develop our internal testing project and published as soon as possible.

Reply:
Would you like to develop directly in utPLSQL/utPLSQL-maven-plugin project?
There are no active developer for this one at the moment.
Can you join our Slack for utPLSQL (there is java channel)

Reply:
For sure, no problem.